css/style.css

```diff
@@ -0,0 +1,169 @@
+/* all styles */
```

Comment:
There's no need to put css files to the rep. They should be ignored, while you have to provide a setup-ready repository so I could build it locally.